rust/src/adapter/otel_formatter/mod.rs

```diff
@@ -142,7 +142,7 @@ impl Span {
     ) -> Span {
         Span {
             trace_id,
-            span_id: new_span_id().to_hex_16(),
+            span_id: new_span_id().to_hex_8(),
```

Comment:
I realized that these were wrong. span-ids should be 8byte hex and trace ids should be 16byte hex. https://opentelemetry.io/docs/specs/otel/trace/api/#spancontext